internal/catalog/exports.go

```diff
@@ -43,6 +44,7 @@ var (
 	HealthStatusV1Alpha1Type     = types.HealthStatusV1Alpha1Type
 	HealthChecksV1Alpha1Type     = types.HealthChecksV1Alpha1Type
 	DNSPolicyV1Alpha1Type        = types.DNSPolicyV1Alpha1Type
+	FailoverPolicyV1Alpha1Type   = types.FailoverPolicyV1Alpha1Type
```

Comment:
I've found with writing controllers in the mesh package that need catalog resources that it's also good to add the latest version to exports. That way controllers always use the latest version rather than being pinned to a specific version.